sapi/cli/php_cli_server.c

```diff
@@ -2432,6 +2441,24 @@ static char *php_cli_server_parse_addr(const char *addr, int *pport) {
 	return pestrndup(addr, end - addr, 1);
 }
 
+#if defined(HAVE_PRCTL) || defined(HAVE_PROCCTL)
+static void php_cli_server_worker_install_pdeathsig(void)
```

Comment:
A possible evolution might be possible later on for Windows (ie job objects I believe), but letting @cmb69 's expertise confirming whether it s relevant or not.

Reply:
Well, I guess we would first need to support built-in server workers on Windows in the first place (probably requires a multi-threaded implementation). Unless we have that, I don't see much point in adding partial Windows support at all.

Reply:
Right, Windows has no fork. This would likely have to be done via threads which don't suffer from this problem (killing the process will stop all threads).